blog/rslint-a-extremely-fast-javascript-linter.md

```diff
@@ -0,0 +1,77 @@
+# RSLint - A extremely fast JavaScript linter
+
+ESLint is a de facto solution for [linting](https://en.wikipedia.org/wiki/Lint_%28software%29) JavaScript code. Also, `C` is the first that comes to our minds when we talk about compile some code from every kind. I know that a lot of compilers are compiled using the language that them compiles 😕. But when, we talk about parsers, abstract syntax trees (AST), and compile to binary: `C` comes to our minds. Maybe I'm old, I don't know. 🤔
+
+And, in this post I'll talk about another linters for JavaScript that are growing around today.
+
+## The Deno alternative
+
+Deno has re thinked the Node.js world with lots of really great decisions, a built-in bundler, documentation generator, a code formatter, and 🥁... [deno_lint](https://github.com/denoland/deno_lint), a linter made in Rust 🦾.
```

Comment:
Deno has re thought the Node.js world with lots of really great decisions: a built-in bundler, documentation generator, a code formatter, and 🥁... [deno_lint](https://github.com/denoland/deno_lint), a linter made in Rust 🦾.